make_call.py

```diff
@@ -0,0 +1,20 @@
+# Download the helper library from https://www.twilio.com/docs/python/install
+import os
+from twilio.rest import Client
+
+
+
+# Find your Account SID and Auth Token at twilio.com/console
+# and set the environment variables. See http://twil.io/secure
+account_sid = 'AC4ccd7e15f30bc287ef5bb019cedd0c0a'
+auth_token = '91b22def36f592eb2bef7c56eb62b237'
```

Comment:
Please remove your personal token and sid otherwise it's going to run up your bill substantially :)

Reply:
@thalex  Dude remove your keys or change the token man